app/models/concerns/has_project_role.rb

```diff
@@ -0,0 +1,13 @@
+module HasProjectRole
+  extend ActiveSupport::Concern
```

Comment:
Good point. When I wrote this I was still not aware of the differences between a regular module and a module extending `ActiveSupport::Concern`, and ended up following the same example we already had for `Role`.

I'll take it out. Thanks for spotting it.